src/app/src/store/authSlice.js

```diff
@@ -20,9 +22,21 @@ export const authSlice = createSlice({
                 state.locationBeforeAuth = location;
             }
         },
+        setUtilities: (state, { payload: utilities }) => {
+            state.utilities = utilities;
+        },
+        setSelectedUtility: (state, { payload: utility }) => {
+            state.selectedUtility = utility;
```

Comment:
Consider making this more consistent:

```suggestion
        setSelectedUtility: (state, { payload: selectedUtility }) => {
            state.selectedUtility = selectedUtility;
```